src/App.css

```diff
@@ -32,6 +32,7 @@
   from {
     transform: rotate(0deg);
   }
+
```

Comment:
### Optional
- [ ] It would be better if we remove the following since they are not part of the project; `logo192.png logo512.png manifest.json, robots.txt`. Also, remember to remove all paths linked to